scripts/checkServicesTasks.js

```diff
@@ -0,0 +1,122 @@
+const RedisSMQ = require('rsmq');
+const Redis = require('redis');
+
+const { tenant, redisUrl, read, deleteAll, service, deleteOne } = require('yargs')
+  .option('tenant', {
+    alias: 't',
+    type: 'string',
+    describe: 'Tenant to check',
+  })
+  .option('redisUrl', {
+    alias: 'u',
+    type: 'string',
+    describe: 'Redis url',
+    default: 'redis://localhost:6379',
+  })
+  .option('service', {
+    alias: 's',
+    type: 'string',
+    describe: 'Service queue',
+    choices: ['information_extraction', 'segmentation'],
+  })
+  .option('read', {
+    alias: 'r',
+    type: 'boolean',
+    describe: 'Read the queue',
+    default: true,
+  })
+  .option('deleteAll', {
+    alias: 'D',
+    type: 'boolean',
+    describe: 'Delete all messages from the queue',
+    default: false,
+  })
+  .option('deleteOne', {
+    alias: 'd',
+    type: 'string',
+    describe: 'ID of the message to be deleted',
+  })
+
+  .demandOption(['service'], '\n\n').argv;
+
+const run = async () => {
+  const redisClient = await Redis.createClient(redisUrl);
+  const redisSMQ = await new RedisSMQ({ client: redisClient });
+
+  const readFirstTaskMessage = async () => {
+    const message = await redisSMQ.receiveMessageAsync({
+      qname: `${service}_tasks`,
+      vt: 1,
+    });
+
+    if (!message.id) {
+      return undefined;
+    }
+
+    return message;
+  };
+
+  const deleteMessage = async id => {
+    await redisSMQ.deleteMessageAsync({
+      qname: `${service}_tasks`,
+      id,
+    });
+  };
+
+  const readAllTaskMessages = async () => {
+    const messages = [];
+    while (true) {
```

Comment:
how about something like:
```
let message = await readFirstTaskMessage();
while(message){
messages.push(message);
message = await readFirstTaskMessage();
}
```
This avoids the constant in the loop warning from eslint.